ckanext/canada/templates/package/wet_datatable.html

```diff
@@ -99,6 +99,40 @@ <h2 id="prtable">{{_("Preview")}}</h2>
               }));
             });
             form.appendTo('body').submit();
+        }},
+        {extend:"selected",text:"{{_('Delete records')}}",action: function ( e, dt, button, config ){
+            // closing form tag needed for py webtest lib
+            var form = $('<form></form>',{
+              action : "{{ h.url_for('canada.delete_selected_records', resource_id=resource_id) }}",
+              method : "post"
+            });
+            form.append($('<input>').attr({
+              type: 'hidden',
+              value: "{{resource_name}}",
+              name: 'resource_name'
+            }));
+            form.append($('<input>').attr({
+              type: 'hidden',
+              value: "{{request.cookies.token}}",
+              name: 'token'
+            }));
+            form.append($('<input>').attr({
+              type: 'hidden',
+              value: {{primary_keys}},
```

Comment:
Add `| tojson` filter for the `primary_keys`, which solves any encoding issues with Jinja2